components/widgets/team_icon/team_icon.tsx

```diff
@@ -37,9 +37,6 @@ export class TeamIcon extends React.PureComponent<Props> {
 
     public render() {
         const {name, url, size, withHover} = this.props;
-        if (!name && !url) {
```

Comment:
With this change we are allowing to show a team icon without an url or a team name, so... we wont be able to show anything